### ISSUE TITLE
Kill daemon if it's not going down gracefully

### ISSUE THREAD

**Issue by ghost** [jnrbsn/daemonocle#9]
In [stop action](https://github.com/jnrbsn/daemonocle/blob/master/daemonocle/daemon.py#L462) it tries to terminate process with SIGTERM and if it fails just shows a message, but sometimes daemons aren't peaceful, might not even listen to signals or busy with their evil actions so we have to use force to get rid of them.
I just wanted to go with some sort of another try except block with `SIGKILL` but I thought I might not keep up with style and other stuff, would you add it please?

Thanks.

**Comment by jnrbsn** [comment]
@k1-hedayati: First, I apologize for the extremely delayed response. I have been working on version 1.0 of daemonocle, which I'm going to release very soon. This new version has many bug fixes as well as a test suite with over 90% code coverage. With the new continuous integration I've setup, I'll be able to more quickly and easily respond to feature requests like these. With that said, I'm considering your feature request for version 1.1, which I will start working on next.

**Comment by k1-hedayati** [comment]
Thanks :+1:

**Comment by jnrbsn** [comment]
This functionality is now available in version 1.1.0. You can pass the `force` argument to `do_action` like this:

```python
daemon.do_action('stop', force=True)
```

Or if you're using the CLI via click, you can just do this:

```
$ python example.py stop --force
```
